### ISSUE TITLE
[Fwa

### ISSUE THREAD

**Issue by Ultra980** [nhirokinet/bluelineconsole#244]

**Comment by rokejulianlockhart** [comment]
Yeah, F-Droid supports reproducible builds, so signature mismatch shouldn't be a problem if the extension were distributed via there (if the Play Store's policies are problematic, that is).
